recipe/meta.yaml

```
@@ -1,34 +1,44 @@
 {% set name = "numpyro" %}
-{% set version = "0.11.0" %}
+{% set version = "0.15.3" %}
 
 package:
-  name: numpyro
+  name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: d357dd1a0159f86d45c233708ad33a3d3d53431d05b4bbece1d068ae4168aa8e
+  sha256: f445eeae6200f883d790d65ce29ff245f7a9639bac3322d993eccf91c44023b3
 
 build:
-  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
   number: 0
-  script: {{ PYTHON }} -m pip install .
+  # Missing jax and jaxlib python version 3.9
+  # https://github.com/jax-ml/jax/blob/jax-v0.4.35/setup.py#L118
+  # Missing jax and jaxlib for s390x
+  skip: True  # [py<=39 or py>311 or s390x]
```

```suggestion
  skip: True  # [py<=39 or s390x]
```

`jax/jaxlib` are available for `py312`

---

But numpyro does not support 3.12.
https://github.com/pyro-ppl/numpyro/blob/master/setup.py#L102

---

that is metadata, the actual constrain is this one https://github.com/pyro-ppl/numpyro/blob/0.15.3/setup.py#L87

---

if it does not build, then we skip py312 with a comment

---

@Andrii-Osipov